fsp_email.py

```diff
@@ -87,6 +87,22 @@ def parse_date(tz, m):
         raise UnusableEmail("Unable to convert %s to UTC: %r" % (local, e))
     return z.replace(tzinfo=None)
 
+def monthToNum(shortMonth):
+    return{
```

Comment:
PEP-8 and standard Python [style nits](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements): insert space between `return` and `{`, and delete spaces before colons.